### PR TITLE
Update GetInteriorHeading.md

### PR DESCRIPTION
**Pull request by freedy69**
Added information about the return value.

### CODE REVIEW

File: INTERIOR/GetInteriorHeading.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-NativeDB Introduced: v1493
+float GET_INTERIOR_HEADING(int interior);
 ```
 
 ## Parameters
-* **interior**:
+* **interior**: ID of the interior
+
+## Return value
+Returns interior heading in radians. Multiply the returned value with 57.29578 to get it in degrees.
```
Comment (4mmonium) [inline INTERIOR/GetInteriorHeading.md:15]: Just a quick note, that `57.29578` is the result of [180/Pi](https://en.wikipedia.org/wiki/Pi) or [360/τ](https://en.wikipedia.org/wiki/Turn_(angle)#Tau_proposals) for people that prefer such.

### Pi = π (3.14159...)
### Tau = 2π (6.28318...)

**Review by 4mmonium — CHANGES_REQUESTED** [review]
Hey almost there, you forgot to add the alias for the native since you changed the function name 😊

File: INTERIOR/GetInteriorHeading.md (hunk 1 above)
Comment (freedy69) [reply INTERIOR/GetInteriorHeading.md]: I changed the return value text, how is it now?

Comment (4mmonium) [reply INTERIOR/GetInteriorHeading.md]: > I changed the return value text, how is it now?

LGTM now, thank you 👍

**Review by 4mmonium — APPROVED** [review]

**Pull request merged**